client/src/components/DatePicker/DatePicker.css

```diff
@@ -0,0 +1,29 @@
+@font-face {
+  font-family: Raleway;
+  src: url("https://fonts.googleapis.com/css?family=Raleway&display=swap");
+}
+
+label {
```

Comment:
Can you use classes for these? If you select these with just the tags, then these styles will clash with the styles in other components

Reply:
Done